include/YTArchive.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import unicode_literals; import youtube_dl
+
+class MyLogger(object): # prints out warning and error messages
+    def debug(self, msg): pass
+    def warning(self, msg): print(msg)
+    def error(self, msg): print(msg)
+def my_hook(d):
+    if d['status'] == 'finished': print('Done downloading... start converting')  
+def downer(code, link): # standard video and playlist specs for Hydrus
```

Comment:
Nondescript function name and parameter names

Reply:
Every video specification has its own code reference for YouTube